src/foam/u2/FoamTagLoader.js

```diff
@@ -57,4 +57,4 @@ foam.CLASS({
   ]
 });
 
-foam.u2.FoamTagLoader.create();
+//foam.u2.FoamTagLoader.create();
```

Comment:
re: FoamTagLoader
Users/jhughes/workspace/jbbmobile/repos/sefoam-quote/build/foam-java-bin.js:32589
      this.window.addEventListener('load', this.onLoad, false);
                  ^

TypeError: this.window.addEventListener is not a function

flags: ['web']

other 'web' files are needed, so you would need an additional flag.